app/scripts/pages/files.js

```diff
@@ -63,14 +59,12 @@ export default React.createClass({
   onDragLeave: function (e) {
     console.log('dragleave')
     this.setState({ dragging: false })
-    $(e.target).removeClass('hover')
     e.stopPropagation()
     e.preventDefault()
   },
 
   onDrop: function (e) {
     this.setState({ dragging: false })
-    $(e.target).removeClass('hover')
```

Comment:
The only css the defiend a `hover` class was for an element matching the following selector `.file-add-container .file-add-container-inner` which none of these effected. The `hover` class for that is handled on line `154` (old code) or `148` (new code).